cves/kernel/CVE-2017-6074.yml

```diff
@@ -278,8 +298,11 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The vulnerability was due to the freeing of the memory of a socket buffer 
+    in the Datagram Congestion Control Protocol (DCCP). Additionally, this 
```

Comment:
I would include a more in depth explanation of what DCCP is somewhere in this document